armbian/base/scripts/bbb-config.sh

```diff
@@ -300,7 +300,7 @@ case "${COMMAND}" in
 
     exec)
         case "${SETTING}" in
-            BITCOIN_REINDEX)
+            BITCOIN_REINDEX|BITCOIN_RESYNC)
```

Comment:
Can you add the string to help at the top as well?

Reply:
Good point, done.